src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -55,14 +57,16 @@ export default class GenomicOverview extends React.Component<IGenomicOverviewPro
         return (
             <div style={{ display:'flex' }}>
                 <Tracks mutations={this.props.mutations} key={Math.random()} sampleManager={this.props.sampleManager} width={this.props.getWidth()} cnaSegments={this.props.cnaSegments} />
-                <ThumbnailExpandVAFPlot
-                    data={this.state.frequencies}
-                    order={this.props.sampleManager.sampleIndex}
-                    colors={this.props.sampleColors}
-                    labels={labels}
-                    overlayPlacement="right"
-                    cssClass="vafPlot"
-                />
+                <If condition={this.props.mutations.length > 0}>
```

Comment:
I'm not sure how I feel. On one hand I like this more. On the other hand I actually think in general it makes it slightly less readable, because you need to match up the `Else` and `If`. But in this case it's just the simple <If>. So I don't really care either way, was just curious.